tools/extra/fpgadiag/fpgastats.py

```diff
@@ -169,7 +171,7 @@ def main():
     f = FpgaFinder(args.segment, args.bus, args.device, args.function)
     devs = f.find()
     for d in devs:
-        print('bdf: {segment:04x}:{bus:02x}:{dev:02x}.{func:x}'.format(**d))
+        print('bdf: {segment:04x}:{bus:02x}:{dev:02x}.{func:x}'.format(*d))
```

Comment:
Just ran it. I don't think I was right with the one*. Seems to be erroring out. Can you revert it back? Sorry.

Reply:
reverted